packages/edit-site/src/components/header-edit-mode/document-actions/index.js

```diff
@@ -157,34 +157,38 @@ export default function DocumentActions() {
 					{ label ?? '' }
 				</Text>
 
-				<Dropdown
-					popoverProps={ popoverProps }
-					position="bottom center"
-					renderToggle={ ( { isOpen, onToggle } ) => (
-						<Button
-							className="edit-site-document-actions__get-info"
-							icon={ chevronDown }
-							aria-expanded={ isOpen }
-							aria-haspopup="true"
-							onClick={ onToggle }
-							variant={ showIconLabels ? 'tertiary' : undefined }
-							label={ sprintf(
-								/* translators: %s: the entity to see details about, like "template"*/
-								__( 'Show %s details' ),
-								entityLabel
-							) }
-						>
-							{ showIconLabels && __( 'Details' ) }
-						</Button>
-					) }
-					contentClassName="edit-site-document-actions__info-dropdown"
-					renderContent={ ( { onClose } ) => (
-						<TemplateDetails
-							template={ template }
-							onClose={ onClose }
-						/>
-					) }
-				/>
+				{ showDropdown && (
```

Comment:
No change in this file apart from this new prop to allow showing this component without the dropdown.

Reply:
Do we need this change? I'd guess it's a leftover for showing that in the `view` mode? 🤔

Reply:
Yes, I think it's not needed anymore